2015-12-15 13:29:07 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Admin could not be found.
Exception Attributes: array (
  'class' => 'Admin',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /s_n_clone/social_networking/admin
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-12-15 13:29:17 Error: [Cake\Controller\Exception\MissingActionException] Action UsersController::admin() could not be found, or is not accessible.
Exception Attributes: array (
  'controller' => 'UsersController',
  'action' => 'admin',
  'prefix' => '',
  'plugin' => NULL,
)
Request URL: /s_n_clone/social_networking/users/admin
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#1 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#2 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 {main}


2015-12-15 13:29:27 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Admin could not be found.
Exception Attributes: array (
  'class' => 'Admin',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /s_n_clone/social_networking/admin/users
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-12-15 14:07:45 Warning: Warning (4096): Argument 1 passed to App\Controller\UsersController::beforeFilter() must be an instance of App\Controller\Event, instance of Cake\Event\Event given, called in C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventManager.php on line 385 and defined in [C:\xampp\htdocs\s_n_clone\social_networking\src\Controller\UsersController.php, line 15]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 140
App\Controller\UsersController::beforeFilter() - APP/Controller\UsersController.php, line 15
Cake\Event\EventManager::_callListener() - CORE\src\Event\EventManager.php, line 385
Cake\Event\EventManager::dispatch() - CORE\src\Event\EventManager.php, line 355
Cake\Controller\Controller::dispatchEvent() - CORE\src\Event\EventDispatcherTrait.php, line 78
Cake\Controller\Controller::startupProcess() - CORE\src\Controller\Controller.php, line 491
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 109
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-12-15 14:07:45 Error: Fatal Error (1): Call to a member function allow() on boolean in [C:\xampp\htdocs\s_n_clone\social_networking\src\Controller\UsersController.php, line 18]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 209
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 92
[main] - [internal], line ??



2015-12-15 14:07:46 Error: [Cake\Error\FatalErrorException] Call to a member function allow() on boolean
Request URL: /s_n_clone/social_networking/users/add
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(92): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 18)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-12-15 14:08:22 Warning: Warning (4096): Argument 1 passed to App\Controller\UsersController::beforeFilter() must be an instance of App\Controller\Event, instance of Cake\Event\Event given, called in C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventManager.php on line 385 and defined in [C:\xampp\htdocs\s_n_clone\social_networking\src\Controller\UsersController.php, line 15]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 140
App\Controller\UsersController::beforeFilter() - APP/Controller\UsersController.php, line 15
Cake\Event\EventManager::_callListener() - CORE\src\Event\EventManager.php, line 385
Cake\Event\EventManager::dispatch() - CORE\src\Event\EventManager.php, line 355
Cake\Controller\Controller::dispatchEvent() - CORE\src\Event\EventDispatcherTrait.php, line 78
Cake\Controller\Controller::startupProcess() - CORE\src\Controller\Controller.php, line 491
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 109
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-12-15 14:09:39 Warning: Warning (4096): Argument 1 passed to App\Controller\UsersController::beforeFilter() must be an instance of App\Controller\Event, instance of Cake\Event\Event given, called in C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventManager.php on line 385 and defined in [C:\xampp\htdocs\s_n_clone\social_networking\src\Controller\UsersController.php, line 15]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 140
App\Controller\UsersController::beforeFilter() - APP/Controller\UsersController.php, line 15
Cake\Event\EventManager::_callListener() - CORE\src\Event\EventManager.php, line 385
Cake\Event\EventManager::dispatch() - CORE\src\Event\EventManager.php, line 355
Cake\Controller\Controller::dispatchEvent() - CORE\src\Event\EventDispatcherTrait.php, line 78
Cake\Controller\Controller::startupProcess() - CORE\src\Controller\Controller.php, line 491
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 109
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-12-15 14:09:39 Error: Fatal Error (1): Call to a member function allow() on boolean in [C:\xampp\htdocs\s_n_clone\social_networking\src\Controller\UsersController.php, line 18]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 209
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 92
[main] - [internal], line ??



2015-12-15 14:09:39 Error: [Cake\Error\FatalErrorException] Call to a member function allow() on boolean
Request URL: /s_n_clone/social_networking/users/add
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(92): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 18)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-12-15 14:10:30 Error: Fatal Error (1): Call to a member function allow() on boolean in [C:\xampp\htdocs\s_n_clone\social_networking\src\Controller\UsersController.php, line 19]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 209
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 92
[main] - [internal], line ??



2015-12-15 14:10:31 Error: [Cake\Error\FatalErrorException] Call to a member function allow() on boolean
Request URL: /s_n_clone/social_networking/users/add
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(92): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 19)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-12-15 14:21:38 Error: [Cake\Controller\Exception\MissingActionException] Action UsersController::login() could not be found, or is not accessible.
Exception Attributes: array (
  'controller' => 'UsersController',
  'action' => 'login',
  'prefix' => '',
  'plugin' => NULL,
)
Request URL: /s_n_clone/social_networking/users/login
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#1 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#2 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 {main}


2015-12-15 14:22:58 Error: [Cake\Controller\Exception\MissingActionException] Action UsersController::login() could not be found, or is not accessible.
Exception Attributes: array (
  'controller' => 'UsersController',
  'action' => 'login',
  'prefix' => '',
  'plugin' => NULL,
)
Request URL: /s_n_clone/social_networking/users/login
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#1 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#2 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 {main}


2015-12-15 14:24:12 Error: [Cake\View\Exception\MissingTemplateException] Template file "Users\login.ctp" is missing.
Exception Attributes: array (
  'file' => 'Users\\login.ctp',
)
Request URL: /s_n_clone/social_networking/users/login
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\View\View.php(576): Cake\View\View->_getViewFileName(NULL)
#1 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Controller.php(611): Cake\View\View->render(NULL, NULL)
#2 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(120): Cake\Controller\Controller->render()
#3 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#4 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#5 {main}


2015-12-15 15:15:21 Error: Fatal Error (4): syntax error, unexpected 'echo' (T_ECHO) in [C:\xampp\htdocs\s_n_clone\social_networking\src\Template\Layout\default.ctp, line 40]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 209
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 92
[main] - [internal], line ??



2015-12-15 15:15:22 Error: [Cake\Error\FatalErrorException] syntax error, unexpected 'echo' (T_ECHO)
Request URL: /s_n_clone/social_networking/users/login
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(92): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 40)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-12-15 15:23:43 Error: [Cake\View\Exception\MissingHelperException] Helper class AuthHelper could not be found.
Exception Attributes: array (
  'class' => 'AuthHelper',
  'plugin' => NULL,
)
Request URL: /s_n_clone/social_networking/users/login
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Core\ObjectRegistry.php(91): Cake\View\HelperRegistry->_throwMissingClassError('Auth', NULL)
#1 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\View\HelperRegistry.php(66): Cake\Core\ObjectRegistry->load('Auth')
#2 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\View\View.php(867): Cake\View\HelperRegistry->__isset('Auth')
#3 C:\xampp\htdocs\s_n_clone\social_networking\src\Template\Layout\default.ctp(45): Cake\View\View->__get('Auth')
#4 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\View\View.php(971): include('C:\\xampp\\htdocs...')
#5 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\View\View.php(931): Cake\View\View->_evaluate('C:\\xampp\\htdocs...', Array)
#6 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\View\View.php(625): Cake\View\View->_render('C:\\xampp\\htdocs...')
#7 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\View\View.php(584): Cake\View\View->renderLayout('', 'default')
#8 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Controller.php(611): Cake\View\View->render(NULL, NULL)
#9 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(120): Cake\Controller\Controller->render()
#10 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#11 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#12 {main}


2015-12-15 15:28:52 Error: Fatal Error (1): Call to a member function check() on null in [C:\xampp\htdocs\s_n_clone\social_networking\src\Template\Layout\default.ctp, line 45]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 209
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 92
[main] - [internal], line ??



2015-12-15 15:28:52 Error: [Cake\Error\FatalErrorException] Call to a member function check() on null
Request URL: /s_n_clone/social_networking/users/login
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(92): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 45)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-12-15 15:34:33 Error: Fatal Error (4): syntax error, unexpected '}', expecting ',' or ';' in [C:\xampp\htdocs\s_n_clone\social_networking\src\Template\Layout\default.ctp, line 45]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 209
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 92
[main] - [internal], line ??



2015-12-15 15:34:34 Error: [Cake\Error\FatalErrorException] syntax error, unexpected '}', expecting ',' or ';'
Request URL: /s_n_clone/social_networking/users/login
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(92): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 45)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-12-15 20:32:46 Error: [Cake\Core\Exception\MissingPluginException] Plugin AkkaFacebooks could not be found.
Exception Attributes: array (
  'plugin' => 'AkkaFacebooks',
)
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\config\bootstrap.php(191): Cake\Core\Plugin::load('AkkaFacebooks', Array)
#1 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(27): require('C:\\xampp\\htdocs...')
#2 {main}


2015-12-15 20:50:19 Error: Fatal Error (1): Class 'App\Controller\Router' not found in [C:\xampp\htdocs\s_n_clone\social_networking\src\Controller\AppController.php, line 70]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 209
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 92
[main] - [internal], line ??



2015-12-15 20:50:19 Error: [Cake\Error\FatalErrorException] Class 'App\Controller\Router' not found
Request URL: /s_n_clone/social_networking/users/login
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(92): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Class 'App\\Cont...', 'C:\\xampp\\htdocs...', 70)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-12-15 20:50:37 Error: Fatal Error (1): Class 'App\Controller\Router' not found in [C:\xampp\htdocs\s_n_clone\social_networking\src\Controller\AppController.php, line 70]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 209
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 92
[main] - [internal], line ??



2015-12-15 20:50:37 Error: [Cake\Error\FatalErrorException] Class 'App\Controller\Router' not found
Request URL: /s_n_clone/social_networking/users/login
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(92): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Class 'App\\Cont...', 'C:\\xampp\\htdocs...', 70)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-12-15 20:50:53 Error: Fatal Error (1): Class 'App\Controller\Router' not found in [C:\xampp\htdocs\s_n_clone\social_networking\src\Controller\AppController.php, line 70]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 209
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 92
[main] - [internal], line ??



2015-12-15 20:50:54 Error: [Cake\Error\FatalErrorException] Class 'App\Controller\Router' not found
Request URL: /s_n_clone/social_networking/users/login
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(92): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Class 'App\\Cont...', 'C:\\xampp\\htdocs...', 70)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-12-15 20:52:33 Error: Fatal Error (1): Class 'App\Controller\Router' not found in [C:\xampp\htdocs\s_n_clone\social_networking\src\Controller\AppController.php, line 70]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 209
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 92
[main] - [internal], line ??



2015-12-15 20:52:34 Error: [Cake\Error\FatalErrorException] Class 'App\Controller\Router' not found
Request URL: /s_n_clone/social_networking/users/login
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(92): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Class 'App\\Cont...', 'C:\\xampp\\htdocs...', 70)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-12-15 22:32:02 Error: [Cake\Database\Exception] SQLSTATE[42S02]: Base table or view not found: 1146 Table 'social_networking.facebooks' doesn't exist
Request URL: /s_n_clone/social_networking/users/login?code=AQAZ2e4JQhTVJaVGfzM0vmHYDqEX6yocc3tyuOKkLJm7oj08VJNectinIWuezhoCSWmZy4lCSMH0OEz8xoKzBwV5vHg_Awp7UbV0xT7nzyW-1UfwBF0nZ8dgZnQsEHT5EwIjmXJs2ns2Yl08-aJSt9-gwWDDqpdgM9qpuDJFMMOGwP9EyjagJ6TgZ1_EcYznT9WGeN5ZZsmNK6hJvWXAeGcacSHPi8n4yCMI-jLu1kXbSsMa-hKILTc0sMaQlSxmRFaSaxsnC_kzxHvPXzbHOVJttUX_nNmDEy4BR6qqtKe94Cr7ncKLhFW2NhXvS30ElCG8tYzHJC7J8d6Dnyh240Ak&state=e0f85f8d138788576781c4b17fd150af
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Database\Schema\Collection.php(96): Cake\Database\Schema\Collection->_reflect('Column', 'facebooks', Array, Object(Cake\Database\Schema\Table))
#1 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Database\Schema\CachedCollection.php(64): Cake\Database\Schema\Collection->describe('facebooks', Array)
#2 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(423): Cake\Database\Schema\CachedCollection->describe('facebooks')
#3 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(499): Cake\ORM\Table->schema()
#4 [internal function]: Cake\ORM\Table->primaryKey()
#5 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Association.php(896): call_user_func_array(Array, Array)
#6 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Rule\ExistsIn.php(95): Cake\ORM\Association->__call('primaryKey', Array)
#7 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Rule\ExistsIn.php(95): Cake\ORM\Association\BelongsTo->primaryKey()
#8 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesChecker.php(312): Cake\ORM\Rule\ExistsIn->__invoke(Object(App\Model\Entity\User), Array)
#9 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesChecker.php(312): Cake\Datasource\RulesChecker->Cake\Datasource\{closure}(Object(App\Model\Entity\User), Array)
#10 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesChecker.php(254): Cake\Datasource\RulesChecker->Cake\Datasource\{closure}(Object(App\Model\Entity\User), Array)
#11 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesChecker.php(227): Cake\Datasource\RulesChecker->checkCreate(Object(App\Model\Entity\User), Array)
#12 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesAwareTrait.php(65): Cake\Datasource\RulesChecker->check(Object(App\Model\Entity\User), 'create', Array)
#13 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1420): Cake\ORM\Table->checkRules(Object(App\Model\Entity\User), 'create', Object(ArrayObject))
#14 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1377): Cake\ORM\Table->_processSave(Object(App\Model\Entity\User), Object(ArrayObject))
#15 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Database\Connection.php(557): Cake\ORM\Table->Cake\ORM\{closure}(Object(Cake\Database\Connection))
#16 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1378): Cake\Database\Connection->transactional(Object(Closure))
#17 C:\xampp\htdocs\s_n_clone\social_networking\vendor\akkaweb\cakephp-facebook\src\Controller\Component\GraphComponent.php(390): Cake\ORM\Table->save(Object(App\Model\Entity\User))
#18 C:\xampp\htdocs\s_n_clone\social_networking\vendor\akkaweb\cakephp-facebook\src\Controller\Component\GraphComponent.php(338): AkkaFacebook\Controller\Component\GraphComponent->__newAccount()
#19 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventManager.php(385): AkkaFacebook\Controller\Component\GraphComponent->startup(Object(Cake\Event\Event))
#20 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventManager.php(355): Cake\Event\EventManager->_callListener(Array, Object(Cake\Event\Event))
#21 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventDispatcherTrait.php(78): Cake\Event\EventManager->dispatch(Object(Cake\Event\Event))
#22 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Controller.php(495): Cake\Controller\Controller->dispatchEvent('Controller.star...')
#23 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(109): Cake\Controller\Controller->startupProcess()
#24 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#25 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#26 {main}


2015-12-15 22:37:39 Error: [Cake\Database\Exception] SQLSTATE[42S02]: Base table or view not found: 1146 Table 'social_networking.facebooks' doesn't exist
Request URL: /s_n_clone/social_networking/users/login?code=AQBLuaHpBORcMvasWG78ah4epB7UMHJCc-poCohH9egg41j2fcsO30lNIoG3ojnO9g_2wIuCSdbHWzr5BRmX40VgW97ECpROSVocmdNhyzDV8dOnCHCNjiqtRaEc9anA4xBudgHa4Kx3EhhE2bJTljt6eY732TBXS7lfPA9B5NHSD80RxmloVPDiU7vY-FHL4EXyyf4DWtrCLHgVJgKVTjW_H8cZ8jHKfxf8PMTkJfD5zxYfGjC2qNzntYlXJgGvBdzPw6fqpRgxpZscuossP1V20U3uolhZdUhxhjzCssQ-M-pRUALRa9TlQcdQ1nYA8Tg&state=bd2a8c9d406c66a3dcc92f0587c6990d
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Database\Schema\Collection.php(96): Cake\Database\Schema\Collection->_reflect('Column', 'facebooks', Array, Object(Cake\Database\Schema\Table))
#1 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Database\Schema\CachedCollection.php(64): Cake\Database\Schema\Collection->describe('facebooks', Array)
#2 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(423): Cake\Database\Schema\CachedCollection->describe('facebooks')
#3 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(499): Cake\ORM\Table->schema()
#4 [internal function]: Cake\ORM\Table->primaryKey()
#5 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Association.php(896): call_user_func_array(Array, Array)
#6 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Rule\ExistsIn.php(95): Cake\ORM\Association->__call('primaryKey', Array)
#7 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Rule\ExistsIn.php(95): Cake\ORM\Association\BelongsTo->primaryKey()
#8 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesChecker.php(312): Cake\ORM\Rule\ExistsIn->__invoke(Object(App\Model\Entity\User), Array)
#9 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesChecker.php(312): Cake\Datasource\RulesChecker->Cake\Datasource\{closure}(Object(App\Model\Entity\User), Array)
#10 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesChecker.php(254): Cake\Datasource\RulesChecker->Cake\Datasource\{closure}(Object(App\Model\Entity\User), Array)
#11 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesChecker.php(227): Cake\Datasource\RulesChecker->checkCreate(Object(App\Model\Entity\User), Array)
#12 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesAwareTrait.php(65): Cake\Datasource\RulesChecker->check(Object(App\Model\Entity\User), 'create', Array)
#13 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1420): Cake\ORM\Table->checkRules(Object(App\Model\Entity\User), 'create', Object(ArrayObject))
#14 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1377): Cake\ORM\Table->_processSave(Object(App\Model\Entity\User), Object(ArrayObject))
#15 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Database\Connection.php(557): Cake\ORM\Table->Cake\ORM\{closure}(Object(Cake\Database\Connection))
#16 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1378): Cake\Database\Connection->transactional(Object(Closure))
#17 C:\xampp\htdocs\s_n_clone\social_networking\vendor\akkaweb\cakephp-facebook\src\Controller\Component\GraphComponent.php(386): Cake\ORM\Table->save(Object(App\Model\Entity\User))
#18 C:\xampp\htdocs\s_n_clone\social_networking\vendor\akkaweb\cakephp-facebook\src\Controller\Component\GraphComponent.php(334): AkkaFacebook\Controller\Component\GraphComponent->__newAccount()
#19 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventManager.php(385): AkkaFacebook\Controller\Component\GraphComponent->startup(Object(Cake\Event\Event))
#20 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventManager.php(355): Cake\Event\EventManager->_callListener(Array, Object(Cake\Event\Event))
#21 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventDispatcherTrait.php(78): Cake\Event\EventManager->dispatch(Object(Cake\Event\Event))
#22 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Controller.php(495): Cake\Controller\Controller->dispatchEvent('Controller.star...')
#23 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(109): Cake\Controller\Controller->startupProcess()
#24 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#25 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#26 {main}


2015-12-15 22:39:07 Warning: Warning (4096): Object of class Facebook\GraphUser could not be converted to string in [C:\xampp\htdocs\s_n_clone\social_networking\vendor\akkaweb\cakephp-facebook\src\Controller\Component\GraphComponent.php, line 246]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 140
AkkaFacebook\Controller\Component\GraphComponent::beforeFilter() - ROOT\vendor\akkaweb\cakephp-facebook\src\Controller\Component\GraphComponent.php, line 246
Cake\Event\EventManager::_callListener() - CORE\src\Event\EventManager.php, line 385
Cake\Event\EventManager::dispatch() - CORE\src\Event\EventManager.php, line 355
Cake\Controller\Controller::dispatchEvent() - CORE\src\Event\EventDispatcherTrait.php, line 78
Cake\Controller\Controller::startupProcess() - CORE\src\Controller\Controller.php, line 491
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 109
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-12-15 22:39:28 Error: This authorization code has been used.
2015-12-15 22:39:37 Error: [Cake\Database\Exception] SQLSTATE[42S02]: Base table or view not found: 1146 Table 'social_networking.facebooks' doesn't exist
Request URL: /s_n_clone/social_networking/users/login?code=AQBiLqL0N_BkwrMn0dukr7EwdBAytr4e7DdtD-bJ3NPvu2b5HHbgTuxsi6oyxXO5K2mYeEfSVTSauUSRfLRLU5hAdLjEkzK9ncZSjo_C7LzK1EjOgiQXhr4w4oWA6lSk5LqKaSm7U1z8hgiw7bqwOXcQ_ynVUdvkQkxZbfRkjp9YrtjPtMMAumWairgaMsjv_jNU-9rVNLQa6sGiMgS-lgO7y8k1vd7H-md8UijAmuxHK0LC9o9VWX4EQwJsQZVNNQ6fkyQkAwvcj0NKqFQmb_a55Uxe8v9ktpHIX8bf_xlsZbuEnsHr6hSaZ4D3tbduJQM0dVdpR9sim16g2D3yBERu&state=2c2d724f18f17faf3c02f19eb0689c7d
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Database\Schema\Collection.php(96): Cake\Database\Schema\Collection->_reflect('Column', 'facebooks', Array, Object(Cake\Database\Schema\Table))
#1 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Database\Schema\CachedCollection.php(64): Cake\Database\Schema\Collection->describe('facebooks', Array)
#2 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(423): Cake\Database\Schema\CachedCollection->describe('facebooks')
#3 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(499): Cake\ORM\Table->schema()
#4 [internal function]: Cake\ORM\Table->primaryKey()
#5 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Association.php(896): call_user_func_array(Array, Array)
#6 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Rule\ExistsIn.php(95): Cake\ORM\Association->__call('primaryKey', Array)
#7 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Rule\ExistsIn.php(95): Cake\ORM\Association\BelongsTo->primaryKey()
#8 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesChecker.php(312): Cake\ORM\Rule\ExistsIn->__invoke(Object(App\Model\Entity\User), Array)
#9 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesChecker.php(312): Cake\Datasource\RulesChecker->Cake\Datasource\{closure}(Object(App\Model\Entity\User), Array)
#10 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesChecker.php(254): Cake\Datasource\RulesChecker->Cake\Datasource\{closure}(Object(App\Model\Entity\User), Array)
#11 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesChecker.php(227): Cake\Datasource\RulesChecker->checkCreate(Object(App\Model\Entity\User), Array)
#12 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesAwareTrait.php(65): Cake\Datasource\RulesChecker->check(Object(App\Model\Entity\User), 'create', Array)
#13 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1420): Cake\ORM\Table->checkRules(Object(App\Model\Entity\User), 'create', Object(ArrayObject))
#14 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1377): Cake\ORM\Table->_processSave(Object(App\Model\Entity\User), Object(ArrayObject))
#15 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Database\Connection.php(557): Cake\ORM\Table->Cake\ORM\{closure}(Object(Cake\Database\Connection))
#16 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1378): Cake\Database\Connection->transactional(Object(Closure))
#17 C:\xampp\htdocs\s_n_clone\social_networking\vendor\akkaweb\cakephp-facebook\src\Controller\Component\GraphComponent.php(386): Cake\ORM\Table->save(Object(App\Model\Entity\User))
#18 C:\xampp\htdocs\s_n_clone\social_networking\vendor\akkaweb\cakephp-facebook\src\Controller\Component\GraphComponent.php(334): AkkaFacebook\Controller\Component\GraphComponent->__newAccount()
#19 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventManager.php(385): AkkaFacebook\Controller\Component\GraphComponent->startup(Object(Cake\Event\Event))
#20 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventManager.php(355): Cake\Event\EventManager->_callListener(Array, Object(Cake\Event\Event))
#21 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventDispatcherTrait.php(78): Cake\Event\EventManager->dispatch(Object(Cake\Event\Event))
#22 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Controller.php(495): Cake\Controller\Controller->dispatchEvent('Controller.star...')
#23 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(109): Cake\Controller\Controller->startupProcess()
#24 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#25 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#26 {main}


2015-12-15 22:39:55 Error: This authorization code has been used.
2015-12-15 22:40:02 Error: [Cake\Database\Exception] SQLSTATE[42S02]: Base table or view not found: 1146 Table 'social_networking.facebooks' doesn't exist
Request URL: /s_n_clone/social_networking/users/login?code=AQA5RX3jZ5IaQWNjfa1naPnzsMMb7her4c2tR5o_8UvNzE5tPxGbpZUA_zJcgrzip7ZD1gQz48J1EzpxVqeRfL1Gx0wMl8PiTSvSh9a6ykYu2QGMoeND-w_m7OLu-RDdvO9Yc7bPrUOREXY6CWi4lcVIKk7UZ7dKd5PlZL5SPVKhtYxboSh3bbYBkb6nedvDOWVdodZqYNfIfQ69X0ke_64hcw3AiT2Bdyw9NGMooG_KEp68zIgugfCh2OnFUVszfZ2HjdcbWC4DHYndq_j5m8-uv7mStpymAWm1KQl5hjPEd2ujn6uRThDrM2GWR1m4Lq-vbY8rTMWMyy3cnntlevUd&state=de5fab43ed3587370ba97421da9b53ef
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Database\Schema\Collection.php(96): Cake\Database\Schema\Collection->_reflect('Column', 'facebooks', Array, Object(Cake\Database\Schema\Table))
#1 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Database\Schema\CachedCollection.php(64): Cake\Database\Schema\Collection->describe('facebooks', Array)
#2 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(423): Cake\Database\Schema\CachedCollection->describe('facebooks')
#3 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(499): Cake\ORM\Table->schema()
#4 [internal function]: Cake\ORM\Table->primaryKey()
#5 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Association.php(896): call_user_func_array(Array, Array)
#6 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Rule\ExistsIn.php(95): Cake\ORM\Association->__call('primaryKey', Array)
#7 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Rule\ExistsIn.php(95): Cake\ORM\Association\BelongsTo->primaryKey()
#8 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesChecker.php(312): Cake\ORM\Rule\ExistsIn->__invoke(Object(App\Model\Entity\User), Array)
#9 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesChecker.php(312): Cake\Datasource\RulesChecker->Cake\Datasource\{closure}(Object(App\Model\Entity\User), Array)
#10 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesChecker.php(254): Cake\Datasource\RulesChecker->Cake\Datasource\{closure}(Object(App\Model\Entity\User), Array)
#11 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesChecker.php(227): Cake\Datasource\RulesChecker->checkCreate(Object(App\Model\Entity\User), Array)
#12 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesAwareTrait.php(65): Cake\Datasource\RulesChecker->check(Object(App\Model\Entity\User), 'create', Array)
#13 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1420): Cake\ORM\Table->checkRules(Object(App\Model\Entity\User), 'create', Object(ArrayObject))
#14 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1377): Cake\ORM\Table->_processSave(Object(App\Model\Entity\User), Object(ArrayObject))
#15 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Database\Connection.php(557): Cake\ORM\Table->Cake\ORM\{closure}(Object(Cake\Database\Connection))
#16 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1378): Cake\Database\Connection->transactional(Object(Closure))
#17 C:\xampp\htdocs\s_n_clone\social_networking\vendor\akkaweb\cakephp-facebook\src\Controller\Component\GraphComponent.php(386): Cake\ORM\Table->save(Object(App\Model\Entity\User))
#18 C:\xampp\htdocs\s_n_clone\social_networking\vendor\akkaweb\cakephp-facebook\src\Controller\Component\GraphComponent.php(334): AkkaFacebook\Controller\Component\GraphComponent->__newAccount()
#19 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventManager.php(385): AkkaFacebook\Controller\Component\GraphComponent->startup(Object(Cake\Event\Event))
#20 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventManager.php(355): Cake\Event\EventManager->_callListener(Array, Object(Cake\Event\Event))
#21 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventDispatcherTrait.php(78): Cake\Event\EventManager->dispatch(Object(Cake\Event\Event))
#22 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Controller.php(495): Cake\Controller\Controller->dispatchEvent('Controller.star...')
#23 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(109): Cake\Controller\Controller->startupProcess()
#24 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#25 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#26 {main}


2015-12-15 22:40:33 Error: This authorization code has been used.
2015-12-15 22:41:43 Error: This authorization code has been used.
2015-12-15 22:41:52 Error: [Cake\Database\Exception] SQLSTATE[42S02]: Base table or view not found: 1146 Table 'social_networking.facebooks' doesn't exist
Request URL: /s_n_clone/social_networking/users/login?code=AQBZhZFPn_83ytN5qOFNLQK_URPyVHlVAAItzmfLd5pW-zwFYmvBC_0bZS-hxr8WXqfQukSb5zPjYPtSacguLVSUzruKtc8p5lkUnwuqGr_Ddi96WVVZKWW6j0DI8HZU7oyWsUvjvo2mVNx3FTYsuw2SH3VtZ3nU6DsiicmxhqmFIC3c52nRXTkix5rVT9_AJmKP0ZLm7HcPJFg5V7TjHP30o27XNAR_j22rAfzY57eUXoG-y3kAO-YTI4xWn-YBpQT0ZV8by4FgbpkuJ4xfGCars5yxgu17ymvnVzREN5zKHDLDMXhfEuYG76nEFebUVQa2H_efkXen649V-74FgK2p&state=4714ea1f567deebc154454103ee4916b
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Database\Schema\Collection.php(96): Cake\Database\Schema\Collection->_reflect('Column', 'facebooks', Array, Object(Cake\Database\Schema\Table))
#1 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Database\Schema\CachedCollection.php(64): Cake\Database\Schema\Collection->describe('facebooks', Array)
#2 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(423): Cake\Database\Schema\CachedCollection->describe('facebooks')
#3 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(499): Cake\ORM\Table->schema()
#4 [internal function]: Cake\ORM\Table->primaryKey()
#5 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Association.php(896): call_user_func_array(Array, Array)
#6 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Rule\ExistsIn.php(95): Cake\ORM\Association->__call('primaryKey', Array)
#7 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Rule\ExistsIn.php(95): Cake\ORM\Association\BelongsTo->primaryKey()
#8 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesChecker.php(312): Cake\ORM\Rule\ExistsIn->__invoke(Object(App\Model\Entity\User), Array)
#9 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesChecker.php(312): Cake\Datasource\RulesChecker->Cake\Datasource\{closure}(Object(App\Model\Entity\User), Array)
#10 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesChecker.php(254): Cake\Datasource\RulesChecker->Cake\Datasource\{closure}(Object(App\Model\Entity\User), Array)
#11 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesChecker.php(227): Cake\Datasource\RulesChecker->checkCreate(Object(App\Model\Entity\User), Array)
#12 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesAwareTrait.php(65): Cake\Datasource\RulesChecker->check(Object(App\Model\Entity\User), 'create', Array)
#13 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1420): Cake\ORM\Table->checkRules(Object(App\Model\Entity\User), 'create', Object(ArrayObject))
#14 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1377): Cake\ORM\Table->_processSave(Object(App\Model\Entity\User), Object(ArrayObject))
#15 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Database\Connection.php(557): Cake\ORM\Table->Cake\ORM\{closure}(Object(Cake\Database\Connection))
#16 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1378): Cake\Database\Connection->transactional(Object(Closure))
#17 C:\xampp\htdocs\s_n_clone\social_networking\vendor\akkaweb\cakephp-facebook\src\Controller\Component\GraphComponent.php(385): Cake\ORM\Table->save(Object(App\Model\Entity\User))
#18 C:\xampp\htdocs\s_n_clone\social_networking\vendor\akkaweb\cakephp-facebook\src\Controller\Component\GraphComponent.php(333): AkkaFacebook\Controller\Component\GraphComponent->__newAccount()
#19 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventManager.php(385): AkkaFacebook\Controller\Component\GraphComponent->startup(Object(Cake\Event\Event))
#20 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventManager.php(355): Cake\Event\EventManager->_callListener(Array, Object(Cake\Event\Event))
#21 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventDispatcherTrait.php(78): Cake\Event\EventManager->dispatch(Object(Cake\Event\Event))
#22 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Controller.php(495): Cake\Controller\Controller->dispatchEvent('Controller.star...')
#23 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(109): Cake\Controller\Controller->startupProcess()
#24 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#25 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#26 {main}


2015-12-15 22:42:45 Error: This authorization code has been used.
2015-12-15 22:42:53 Error: [Cake\Database\Exception] SQLSTATE[42S02]: Base table or view not found: 1146 Table 'social_networking.facebooks' doesn't exist
Request URL: /s_n_clone/social_networking/users/login?code=AQBr6Wjw1nbyIeC4oSItxYXOWKdqnT5w8cliPcV0vg_JSLsb4qssySt92zpKV-FDhui6rgZ2d04mpVDjc00EwMoz4oSvJf2uC11nuuKeQwtbGDgeFVScMbCWQDOzbRg38U6APS33vNluNjGBzAQInVvwqjpqCBZm2FxZ-uxMO6zbeTbHePM26CKnZyfgHwLYPoRmZq6GnWM5Dvy8j4jrkGtrKeuFk2jQ7AGIzBpljXVl_-VbpQv94tWBYf_gIbEYjTbSXkWXhEGa806H3vd-yeEZbtUMTllIm7-ecDusPh70oUcBe6w5qCPQ2cSXQ70HGMwg-W0fxAZsdM3FAeIFPxiV&state=e147beb9a581ee2e9624b1bfa8185b76
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Database\Schema\Collection.php(96): Cake\Database\Schema\Collection->_reflect('Column', 'facebooks', Array, Object(Cake\Database\Schema\Table))
#1 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Database\Schema\CachedCollection.php(64): Cake\Database\Schema\Collection->describe('facebooks', Array)
#2 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(423): Cake\Database\Schema\CachedCollection->describe('facebooks')
#3 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(499): Cake\ORM\Table->schema()
#4 [internal function]: Cake\ORM\Table->primaryKey()
#5 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Association.php(896): call_user_func_array(Array, Array)
#6 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Rule\ExistsIn.php(95): Cake\ORM\Association->__call('primaryKey', Array)
#7 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Rule\ExistsIn.php(95): Cake\ORM\Association\BelongsTo->primaryKey()
#8 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesChecker.php(312): Cake\ORM\Rule\ExistsIn->__invoke(Object(App\Model\Entity\User), Array)
#9 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesChecker.php(312): Cake\Datasource\RulesChecker->Cake\Datasource\{closure}(Object(App\Model\Entity\User), Array)
#10 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesChecker.php(254): Cake\Datasource\RulesChecker->Cake\Datasource\{closure}(Object(App\Model\Entity\User), Array)
#11 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesChecker.php(227): Cake\Datasource\RulesChecker->checkCreate(Object(App\Model\Entity\User), Array)
#12 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesAwareTrait.php(65): Cake\Datasource\RulesChecker->check(Object(App\Model\Entity\User), 'create', Array)
#13 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1420): Cake\ORM\Table->checkRules(Object(App\Model\Entity\User), 'create', Object(ArrayObject))
#14 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1377): Cake\ORM\Table->_processSave(Object(App\Model\Entity\User), Object(ArrayObject))
#15 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Database\Connection.php(557): Cake\ORM\Table->Cake\ORM\{closure}(Object(Cake\Database\Connection))
#16 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1378): Cake\Database\Connection->transactional(Object(Closure))
#17 C:\xampp\htdocs\s_n_clone\social_networking\vendor\akkaweb\cakephp-facebook\src\Controller\Component\GraphComponent.php(385): Cake\ORM\Table->save(Object(App\Model\Entity\User))
#18 C:\xampp\htdocs\s_n_clone\social_networking\vendor\akkaweb\cakephp-facebook\src\Controller\Component\GraphComponent.php(333): AkkaFacebook\Controller\Component\GraphComponent->__newAccount()
#19 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventManager.php(385): AkkaFacebook\Controller\Component\GraphComponent->startup(Object(Cake\Event\Event))
#20 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventManager.php(355): Cake\Event\EventManager->_callListener(Array, Object(Cake\Event\Event))
#21 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventDispatcherTrait.php(78): Cake\Event\EventManager->dispatch(Object(Cake\Event\Event))
#22 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Controller.php(495): Cake\Controller\Controller->dispatchEvent('Controller.star...')
#23 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(109): Cake\Controller\Controller->startupProcess()
#24 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#25 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#26 {main}


2015-12-15 22:46:10 Error: This authorization code has been used.
2015-12-15 22:46:19 Error: [Cake\Database\Exception] SQLSTATE[42S02]: Base table or view not found: 1146 Table 'social_networking.facebooks' doesn't exist
Request URL: /s_n_clone/social_networking/users/login?code=AQB5CzeT-3f8rItNE8Ry4XAOjzlV7VcTm5nARVelFDPsKTWzg_wymCUtZhJFHduXbdJeTeFloqOn58kv0FsDkyShWbFD-VcPearskBNBNWXOHGnEBjQop0sCKZ50mSjAHe1zBskfpJva52kdM5ejUrMQbsrcTMR5mZx8ZzZq4i6DCT95a71bbFAGkZDgt_VqNV35izn_xY0yxoyMNpg0HHb_kbSD9Nn0nisRRTj9Rep089klHcoIM70yBNn5LMp7DtEf5GIf79vKnvou-6PuCOwGNAnYYlWxoCtq638omvPK1CK7HgJIeXkED40e_e927QTMe20VwlXSLbdRCADITmhC&state=478d76e45d74bfefdff87b2c5f1f27b2
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Database\Schema\Collection.php(96): Cake\Database\Schema\Collection->_reflect('Column', 'facebooks', Array, Object(Cake\Database\Schema\Table))
#1 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Database\Schema\CachedCollection.php(64): Cake\Database\Schema\Collection->describe('facebooks', Array)
#2 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(423): Cake\Database\Schema\CachedCollection->describe('facebooks')
#3 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(499): Cake\ORM\Table->schema()
#4 [internal function]: Cake\ORM\Table->primaryKey()
#5 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Association.php(896): call_user_func_array(Array, Array)
#6 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Rule\ExistsIn.php(95): Cake\ORM\Association->__call('primaryKey', Array)
#7 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Rule\ExistsIn.php(95): Cake\ORM\Association\BelongsTo->primaryKey()
#8 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesChecker.php(312): Cake\ORM\Rule\ExistsIn->__invoke(Object(App\Model\Entity\User), Array)
#9 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesChecker.php(312): Cake\Datasource\RulesChecker->Cake\Datasource\{closure}(Object(App\Model\Entity\User), Array)
#10 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesChecker.php(254): Cake\Datasource\RulesChecker->Cake\Datasource\{closure}(Object(App\Model\Entity\User), Array)
#11 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesChecker.php(227): Cake\Datasource\RulesChecker->checkCreate(Object(App\Model\Entity\User), Array)
#12 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesAwareTrait.php(65): Cake\Datasource\RulesChecker->check(Object(App\Model\Entity\User), 'create', Array)
#13 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1420): Cake\ORM\Table->checkRules(Object(App\Model\Entity\User), 'create', Object(ArrayObject))
#14 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1377): Cake\ORM\Table->_processSave(Object(App\Model\Entity\User), Object(ArrayObject))
#15 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Database\Connection.php(557): Cake\ORM\Table->Cake\ORM\{closure}(Object(Cake\Database\Connection))
#16 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1378): Cake\Database\Connection->transactional(Object(Closure))
#17 C:\xampp\htdocs\s_n_clone\social_networking\vendor\akkaweb\cakephp-facebook\src\Controller\Component\GraphComponent.php(385): Cake\ORM\Table->save(Object(App\Model\Entity\User))
#18 C:\xampp\htdocs\s_n_clone\social_networking\vendor\akkaweb\cakephp-facebook\src\Controller\Component\GraphComponent.php(333): AkkaFacebook\Controller\Component\GraphComponent->__newAccount()
#19 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventManager.php(385): AkkaFacebook\Controller\Component\GraphComponent->startup(Object(Cake\Event\Event))
#20 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventManager.php(355): Cake\Event\EventManager->_callListener(Array, Object(Cake\Event\Event))
#21 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventDispatcherTrait.php(78): Cake\Event\EventManager->dispatch(Object(Cake\Event\Event))
#22 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Controller.php(495): Cake\Controller\Controller->dispatchEvent('Controller.star...')
#23 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(109): Cake\Controller\Controller->startupProcess()
#24 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#25 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#26 {main}


2015-12-15 22:46:23 Error: This authorization code has been used.
2015-12-15 22:47:26 Error: [Cake\Database\Exception] SQLSTATE[42S02]: Base table or view not found: 1146 Table 'social_networking.facebooks' doesn't exist
Request URL: /s_n_clone/social_networking/users/login?code=AQAqSpaYGmJiXoAxMjkc2G3euP0ziZa9_pVb0WjZNCwhchr9a5gv6qXJtagCmy6PdmY-TE-ZElM45pzOwo1WT5aVWWj5kYGYp-2FXhcGd38wVL9B3Q9McxsrsVq5wC6xP9IUaUTIswyvNC0SuOsVpi3o2n3KlnBDsa6Uw7DJiSLZGkZKJRXTcrRXLb54ONa8YuMMp0r7UoNo1pS6a66aorB7j627YGdDsCtZ8_OcebDZtQizuvAyqnvXcoZPR4770r7mKQsc4hB-gkB4abtnd3pH-i8pCC5hZd0vcla3gvgzzxPoeQxgPSEpoHcewmTqTaFaEFXjjrs7HPb3sFK8wR8d&state=21c5ac1418c74568a2fb385d48c731bd
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Database\Schema\Collection.php(96): Cake\Database\Schema\Collection->_reflect('Column', 'facebooks', Array, Object(Cake\Database\Schema\Table))
#1 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Database\Schema\CachedCollection.php(64): Cake\Database\Schema\Collection->describe('facebooks', Array)
#2 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(423): Cake\Database\Schema\CachedCollection->describe('facebooks')
#3 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(499): Cake\ORM\Table->schema()
#4 [internal function]: Cake\ORM\Table->primaryKey()
#5 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Association.php(896): call_user_func_array(Array, Array)
#6 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Rule\ExistsIn.php(95): Cake\ORM\Association->__call('primaryKey', Array)
#7 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Rule\ExistsIn.php(95): Cake\ORM\Association\BelongsTo->primaryKey()
#8 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesChecker.php(312): Cake\ORM\Rule\ExistsIn->__invoke(Object(App\Model\Entity\User), Array)
#9 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesChecker.php(312): Cake\Datasource\RulesChecker->Cake\Datasource\{closure}(Object(App\Model\Entity\User), Array)
#10 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesChecker.php(254): Cake\Datasource\RulesChecker->Cake\Datasource\{closure}(Object(App\Model\Entity\User), Array)
#11 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesChecker.php(227): Cake\Datasource\RulesChecker->checkCreate(Object(App\Model\Entity\User), Array)
#12 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesAwareTrait.php(65): Cake\Datasource\RulesChecker->check(Object(App\Model\Entity\User), 'create', Array)
#13 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1420): Cake\ORM\Table->checkRules(Object(App\Model\Entity\User), 'create', Object(ArrayObject))
#14 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1377): Cake\ORM\Table->_processSave(Object(App\Model\Entity\User), Object(ArrayObject))
#15 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Database\Connection.php(557): Cake\ORM\Table->Cake\ORM\{closure}(Object(Cake\Database\Connection))
#16 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1378): Cake\Database\Connection->transactional(Object(Closure))
#17 C:\xampp\htdocs\s_n_clone\social_networking\vendor\akkaweb\cakephp-facebook\src\Controller\Component\GraphComponent.php(385): Cake\ORM\Table->save(Object(App\Model\Entity\User))
#18 C:\xampp\htdocs\s_n_clone\social_networking\vendor\akkaweb\cakephp-facebook\src\Controller\Component\GraphComponent.php(333): AkkaFacebook\Controller\Component\GraphComponent->__newAccount()
#19 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventManager.php(385): AkkaFacebook\Controller\Component\GraphComponent->startup(Object(Cake\Event\Event))
#20 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventManager.php(355): Cake\Event\EventManager->_callListener(Array, Object(Cake\Event\Event))
#21 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventDispatcherTrait.php(78): Cake\Event\EventManager->dispatch(Object(Cake\Event\Event))
#22 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Controller.php(495): Cake\Controller\Controller->dispatchEvent('Controller.star...')
#23 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(109): Cake\Controller\Controller->startupProcess()
#24 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#25 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#26 {main}


2015-12-15 22:53:23 Error: [Cake\Database\Exception] SQLSTATE[42S02]: Base table or view not found: 1146 Table 'social_networking.facebooks' doesn't exist
Request URL: /s_n_clone/social_networking/users/login?code=AQD-YwzahlbgJmYMi1EyspMLZg9JHbyxX8O7rChi4ZDpXQ0htEBGty9SoytBZpzyG5O2mnCp7gtX1JRiIkJDGKP1hqT81nfPUbx5vNLV5ecou_WsQcbmPG8cUpYwWd0kK-4bxcKzXXX3Gq-GYG52ewTBbHxc1ARGdVLD6fK03xf3IIBDPZeuGip17H0tZUlzfAAsDk5fVAZJfsz1WoWI6CoxOL5mFpZ3sdhCc4UcZlxIxY0-Qy1hiQx_Y7-HWxiEpjIRcVR8YpLVfHFkYED80SF0gjY4M2kEOszQyVGkE5YJ1akV__CmxVBANW4z4lUX0lU&state=812cd4064246ede91cb049f539e96441
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Database\Schema\Collection.php(96): Cake\Database\Schema\Collection->_reflect('Column', 'facebooks', Array, Object(Cake\Database\Schema\Table))
#1 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Database\Schema\CachedCollection.php(64): Cake\Database\Schema\Collection->describe('facebooks', Array)
#2 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(423): Cake\Database\Schema\CachedCollection->describe('facebooks')
#3 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(499): Cake\ORM\Table->schema()
#4 [internal function]: Cake\ORM\Table->primaryKey()
#5 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Association.php(896): call_user_func_array(Array, Array)
#6 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Rule\ExistsIn.php(95): Cake\ORM\Association->__call('primaryKey', Array)
#7 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Rule\ExistsIn.php(95): Cake\ORM\Association\BelongsTo->primaryKey()
#8 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesChecker.php(312): Cake\ORM\Rule\ExistsIn->__invoke(Object(App\Model\Entity\User), Array)
#9 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesChecker.php(312): Cake\Datasource\RulesChecker->Cake\Datasource\{closure}(Object(App\Model\Entity\User), Array)
#10 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesChecker.php(254): Cake\Datasource\RulesChecker->Cake\Datasource\{closure}(Object(App\Model\Entity\User), Array)
#11 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesChecker.php(227): Cake\Datasource\RulesChecker->checkCreate(Object(App\Model\Entity\User), Array)
#12 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesAwareTrait.php(65): Cake\Datasource\RulesChecker->check(Object(App\Model\Entity\User), 'create', Array)
#13 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1420): Cake\ORM\Table->checkRules(Object(App\Model\Entity\User), 'create', Object(ArrayObject))
#14 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1377): Cake\ORM\Table->_processSave(Object(App\Model\Entity\User), Object(ArrayObject))
#15 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Database\Connection.php(557): Cake\ORM\Table->Cake\ORM\{closure}(Object(Cake\Database\Connection))
#16 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1378): Cake\Database\Connection->transactional(Object(Closure))
#17 C:\xampp\htdocs\s_n_clone\social_networking\vendor\akkaweb\cakephp-facebook\src\Controller\Component\GraphComponent.php(385): Cake\ORM\Table->save(Object(App\Model\Entity\User))
#18 C:\xampp\htdocs\s_n_clone\social_networking\vendor\akkaweb\cakephp-facebook\src\Controller\Component\GraphComponent.php(333): AkkaFacebook\Controller\Component\GraphComponent->__newAccount()
#19 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventManager.php(385): AkkaFacebook\Controller\Component\GraphComponent->startup(Object(Cake\Event\Event))
#20 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventManager.php(355): Cake\Event\EventManager->_callListener(Array, Object(Cake\Event\Event))
#21 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventDispatcherTrait.php(78): Cake\Event\EventManager->dispatch(Object(Cake\Event\Event))
#22 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Controller.php(495): Cake\Controller\Controller->dispatchEvent('Controller.star...')
#23 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(109): Cake\Controller\Controller->startupProcess()
#24 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#25 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#26 {main}


2015-12-15 23:00:49 Error: This authorization code has expired.
2015-12-15 23:01:24 Error: This authorization code has expired.
2015-12-15 23:02:02 Error: This authorization code has expired.
2015-12-15 23:04:06 Error: This authorization code has been used.
2015-12-15 23:04:45 Error: [Cake\Database\Exception] SQLSTATE[42S02]: Base table or view not found: 1146 Table 'social_networking.facebooks' doesn't exist
Request URL: /s_n_clone/social_networking/users/login?code=AQDcF2ddxcf1ePuBPnb_n3tarJkQ_45u4BvPm-gSivz8THPeFgzZjTSURYCLEK3KW6n_Xx6AeQCnyaXQEWYK5CBjzORrtE_c32OUdMdKsK23-L3TKRyaCmrtthHpSKj8VNJOf2-yaunNpkt7HYOKg430fGb8Mj7kb-Dvw1uCTO0Z3uHIIyAgp1aFyRs7YRtAXe1L831_awtGj5PVC8sZSA4Zei8ccLb4-fjRVNI_YcbB3zi2jFszSdhNydKEBjyJNwSYNhUkJPNmdOHJH6BwM3I6d5kbGMvCT_-Z04r2TrR5NRPvMe5VtKPl8dPDEhVZJ_s&state=c5c669015450f6b67a172ee5f5da334d
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Database\Schema\Collection.php(96): Cake\Database\Schema\Collection->_reflect('Column', 'facebooks', Array, Object(Cake\Database\Schema\Table))
#1 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Database\Schema\CachedCollection.php(64): Cake\Database\Schema\Collection->describe('facebooks', Array)
#2 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(423): Cake\Database\Schema\CachedCollection->describe('facebooks')
#3 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(499): Cake\ORM\Table->schema()
#4 [internal function]: Cake\ORM\Table->primaryKey()
#5 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Association.php(896): call_user_func_array(Array, Array)
#6 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Rule\ExistsIn.php(95): Cake\ORM\Association->__call('primaryKey', Array)
#7 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Rule\ExistsIn.php(95): Cake\ORM\Association\BelongsTo->primaryKey()
#8 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesChecker.php(312): Cake\ORM\Rule\ExistsIn->__invoke(Object(App\Model\Entity\User), Array)
#9 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesChecker.php(312): Cake\Datasource\RulesChecker->Cake\Datasource\{closure}(Object(App\Model\Entity\User), Array)
#10 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesChecker.php(254): Cake\Datasource\RulesChecker->Cake\Datasource\{closure}(Object(App\Model\Entity\User), Array)
#11 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesChecker.php(227): Cake\Datasource\RulesChecker->checkCreate(Object(App\Model\Entity\User), Array)
#12 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesAwareTrait.php(65): Cake\Datasource\RulesChecker->check(Object(App\Model\Entity\User), 'create', Array)
#13 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1420): Cake\ORM\Table->checkRules(Object(App\Model\Entity\User), 'create', Object(ArrayObject))
#14 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1377): Cake\ORM\Table->_processSave(Object(App\Model\Entity\User), Object(ArrayObject))
#15 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Database\Connection.php(557): Cake\ORM\Table->Cake\ORM\{closure}(Object(Cake\Database\Connection))
#16 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1378): Cake\Database\Connection->transactional(Object(Closure))
#17 C:\xampp\htdocs\s_n_clone\social_networking\vendor\akkaweb\cakephp-facebook\src\Controller\Component\GraphComponent.php(385): Cake\ORM\Table->save(Object(App\Model\Entity\User))
#18 C:\xampp\htdocs\s_n_clone\social_networking\vendor\akkaweb\cakephp-facebook\src\Controller\Component\GraphComponent.php(333): AkkaFacebook\Controller\Component\GraphComponent->__newAccount()
#19 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventManager.php(385): AkkaFacebook\Controller\Component\GraphComponent->startup(Object(Cake\Event\Event))
#20 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventManager.php(355): Cake\Event\EventManager->_callListener(Array, Object(Cake\Event\Event))
#21 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventDispatcherTrait.php(78): Cake\Event\EventManager->dispatch(Object(Cake\Event\Event))
#22 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Controller.php(495): Cake\Controller\Controller->dispatchEvent('Controller.star...')
#23 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(109): Cake\Controller\Controller->startupProcess()
#24 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#25 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#26 {main}


2015-12-15 23:09:42 Error: [Cake\Database\Exception] SQLSTATE[42S02]: Base table or view not found: 1146 Table 'social_networking.facebooks' doesn't exist
Request URL: /s_n_clone/social_networking/users/login?code=AQBEHtjXhlVf0z5mi9OTZ1itawVYHM-gCXP7aleGQ7jOWu9rj5mGmazvVCyUfSuC1E5tkHr-kqSKJtsmqG1TCuREbrUQgOt7W6S7r9SyaZNOnRyDsa2qZQ4jacEelpEJStZrpkA9pie0Sj5bIrilfSgCeFYxEeegkYBpkCxr5mqM0MYJIXNcrRZjpDgHrAmtQl7G5UaaHtcUfhbQAxlqPvloh0hO5qsrUtiTaOeH6anqd5uqXPykew4CU2iByv-E6LD1nXkuepaLGKX0lAGYyFCouk6Nw78UQXQP_-qWY2qpt1lTQE6LfdWtFICVIO9J_Cg&state=a4f97afc17f07fab08ec8002c0a05f3f
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Database\Schema\Collection.php(96): Cake\Database\Schema\Collection->_reflect('Column', 'facebooks', Array, Object(Cake\Database\Schema\Table))
#1 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Database\Schema\CachedCollection.php(64): Cake\Database\Schema\Collection->describe('facebooks', Array)
#2 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(423): Cake\Database\Schema\CachedCollection->describe('facebooks')
#3 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(499): Cake\ORM\Table->schema()
#4 [internal function]: Cake\ORM\Table->primaryKey()
#5 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Association.php(896): call_user_func_array(Array, Array)
#6 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Rule\ExistsIn.php(95): Cake\ORM\Association->__call('primaryKey', Array)
#7 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Rule\ExistsIn.php(95): Cake\ORM\Association\BelongsTo->primaryKey()
#8 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesChecker.php(312): Cake\ORM\Rule\ExistsIn->__invoke(Object(App\Model\Entity\User), Array)
#9 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesChecker.php(312): Cake\Datasource\RulesChecker->Cake\Datasource\{closure}(Object(App\Model\Entity\User), Array)
#10 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesChecker.php(254): Cake\Datasource\RulesChecker->Cake\Datasource\{closure}(Object(App\Model\Entity\User), Array)
#11 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesChecker.php(227): Cake\Datasource\RulesChecker->checkCreate(Object(App\Model\Entity\User), Array)
#12 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Datasource\RulesAwareTrait.php(65): Cake\Datasource\RulesChecker->check(Object(App\Model\Entity\User), 'create', Array)
#13 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1420): Cake\ORM\Table->checkRules(Object(App\Model\Entity\User), 'create', Object(ArrayObject))
#14 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1377): Cake\ORM\Table->_processSave(Object(App\Model\Entity\User), Object(ArrayObject))
#15 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Database\Connection.php(557): Cake\ORM\Table->Cake\ORM\{closure}(Object(Cake\Database\Connection))
#16 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1378): Cake\Database\Connection->transactional(Object(Closure))
#17 C:\xampp\htdocs\s_n_clone\social_networking\vendor\akkaweb\cakephp-facebook\src\Controller\Component\GraphComponent.php(385): Cake\ORM\Table->save(Object(App\Model\Entity\User))
#18 C:\xampp\htdocs\s_n_clone\social_networking\vendor\akkaweb\cakephp-facebook\src\Controller\Component\GraphComponent.php(333): AkkaFacebook\Controller\Component\GraphComponent->__newAccount()
#19 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventManager.php(385): AkkaFacebook\Controller\Component\GraphComponent->startup(Object(Cake\Event\Event))
#20 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventManager.php(355): Cake\Event\EventManager->_callListener(Array, Object(Cake\Event\Event))
#21 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventDispatcherTrait.php(78): Cake\Event\EventManager->dispatch(Object(Cake\Event\Event))
#22 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Controller.php(495): Cake\Controller\Controller->dispatchEvent('Controller.star...')
#23 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(109): Cake\Controller\Controller->startupProcess()
#24 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#25 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#26 {main}


2015-12-15 23:12:35 Warning: Warning (2): array_combine(): Both parameters should have an equal number of elements in [C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Rule\ExistsIn.php, line 100]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 140
array_combine - [internal], line ??
Cake\ORM\Rule\ExistsIn::__invoke() - CORE\src\ORM\Rule\ExistsIn.php, line 100
Cake\Datasource\RulesChecker::Cake\Datasource\{closure}() - CORE\src\Datasource\RulesChecker.php, line 312
Cake\Datasource\RulesChecker::Cake\Datasource\{closure}() - CORE\src\Datasource\RulesChecker.php, line 312
Cake\Datasource\RulesChecker::checkCreate() - CORE\src\Datasource\RulesChecker.php, line 254
Cake\Datasource\RulesChecker::check() - CORE\src\Datasource\RulesChecker.php, line 227
Cake\ORM\Table::checkRules() - CORE\src\Datasource\RulesAwareTrait.php, line 65
Cake\ORM\Table::_processSave() - CORE\src\ORM\Table.php, line 1420
Cake\ORM\Table::Cake\ORM\{closure}() - CORE\src\ORM\Table.php, line 1377
Cake\Database\Connection::transactional() - CORE\src\Database\Connection.php, line 557
Cake\ORM\Table::save() - CORE\src\ORM\Table.php, line 1378
AkkaFacebook\Controller\Component\GraphComponent::__newAccount() - ROOT\vendor\akkaweb\cakephp-facebook\src\Controller\Component\GraphComponent.php, line 385
AkkaFacebook\Controller\Component\GraphComponent::startup() - ROOT\vendor\akkaweb\cakephp-facebook\src\Controller\Component\GraphComponent.php, line 333
Cake\Event\EventManager::_callListener() - CORE\src\Event\EventManager.php, line 385
Cake\Event\EventManager::dispatch() - CORE\src\Event\EventManager.php, line 355
Cake\Controller\Controller::dispatchEvent() - CORE\src\Event\EventDispatcherTrait.php, line 78
Cake\Controller\Controller::startupProcess() - CORE\src\Controller\Controller.php, line 495
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 109
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-12-15 23:20:34 Error: Fatal Error (1): Call to undefined method AkkaFacebook\Controller\Component\GraphComponent::redirect() in [C:\xampp\htdocs\s_n_clone\social_networking\vendor\akkaweb\cakephp-facebook\src\Controller\Component\GraphComponent.php, line 389]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 209
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 92
[main] - [internal], line ??



2015-12-15 23:20:35 Error: [Cake\Error\FatalErrorException] Call to undefined method AkkaFacebook\Controller\Component\GraphComponent::redirect()
Request URL: /s_n_clone/social_networking/users/login?code=AQDoAyT1zDCgGw9fBsevx2kCTjFAY1ijKpiQwTjSlH7CTr-UQ17X-1B7NTTpbcUz9VQpyGmwbs_o9PH13n1T3uonlUbWP3UDKzVWxKk44hn6IXKb0-L1C2K8R2e80525JexnvzzvGicl3ic0GhjbV5AqUg7p1hdD7165Mnr4OPZK004zy_UIlcrwAP6ye2wuGZoUMntttqGqDnmMYZhYVyEbDzZh4FY6NdPGB5nMr8m6Dai8DfXKt5NkwcoWfGgMwWjR0rx8FaeKrTzWNgDQX17GJHJhVqjIWjxXAyh0q75_d5gyEFbkVzW4Wh66BJL7kGc&state=0973901621956fae6a84dde4ac80706d
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(92): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to undefin...', 'C:\\xampp\\htdocs...', 389)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-12-15 23:23:06 Error: This authorization code has been used.
2015-12-15 23:23:36 Error: [Cake\Datasource\Exception\RecordNotFoundException] Record not found in table "users"
Request URL: /s_n_clone/social_networking/users/login?code=AQAw86oY6vAdid33enXY-ai7oYgO37qsjMNED7s8hkZxheuoVavu0ZwaaWiaWpcj_BvZ5U91A99Y93MMD8WALfW1te0U1mhmcOQ7I8e-3CdL7jq-fJdfUBo-O6-99-iLrSRsLAiaQNr1VonAVumWg6nVag8rgOO_9dvUPKCM7YqvcbECXNpg5JVJWFlDxd16Cvuh483tuYNCUlAyVy316q5b0s9ElTcBEGP0jzxP_BQ01BtMibM72tZCdNNkBL1qUlHB4y64L4IVD14WOQ67Q-ln0mWkC6Ng2MRF4DDvJgxC-1cIuzsQYv_sEAcOMOd-QB8&state=bebd96aef147337d6455dd7d9dc6e414
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1187): Cake\ORM\Query->firstOrFail()
#1 C:\xampp\htdocs\s_n_clone\social_networking\vendor\akkaweb\cakephp-facebook\src\Controller\Component\GraphComponent.php(325): Cake\ORM\Table->get(5)
#2 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventManager.php(385): AkkaFacebook\Controller\Component\GraphComponent->startup(Object(Cake\Event\Event))
#3 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventManager.php(355): Cake\Event\EventManager->_callListener(Array, Object(Cake\Event\Event))
#4 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventDispatcherTrait.php(78): Cake\Event\EventManager->dispatch(Object(Cake\Event\Event))
#5 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Controller.php(495): Cake\Controller\Controller->dispatchEvent('Controller.star...')
#6 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(109): Cake\Controller\Controller->startupProcess()
#7 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#8 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#9 {main}


2015-12-15 23:38:31 Error: Fatal Error (1): Call to a member function build() on boolean in [C:\xampp\htdocs\s_n_clone\social_networking\src\Controller\AppController.php, line 70]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 209
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 92
[main] - [internal], line ??



2015-12-15 23:38:31 Error: [Cake\Error\FatalErrorException] Call to a member function build() on boolean
Request URL: /s_n_clone/social_networking/users/login
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(92): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 70)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-12-15 23:39:12 Error: Fatal Error (1): Call to a member function build() on boolean in [C:\xampp\htdocs\s_n_clone\social_networking\src\Controller\AppController.php, line 65]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 209
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 92
[main] - [internal], line ??



2015-12-15 23:39:13 Error: [Cake\Error\FatalErrorException] Call to a member function build() on boolean
Request URL: /s_n_clone/social_networking/users/login
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(92): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 65)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-12-15 23:39:57 Error: Fatal Error (1): Class 'App\Controller\Router' not found in [C:\xampp\htdocs\s_n_clone\social_networking\src\Controller\AppController.php, line 65]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 209
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 92
[main] - [internal], line ??



2015-12-15 23:39:58 Error: [Cake\Error\FatalErrorException] Class 'App\Controller\Router' not found
Request URL: /s_n_clone/social_networking/users/login
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(92): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Class 'App\\Cont...', 'C:\\xampp\\htdocs...', 65)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-12-15 23:45:24 Error: Fatal Error (1): Call to a member function get_src() on boolean in [C:\xampp\htdocs\s_n_clone\social_networking\src\Controller\AppController.php, line 65]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 209
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 92
[main] - [internal], line ??



2015-12-15 23:45:25 Error: [Cake\Error\FatalErrorException] Call to a member function get_src() on boolean
Request URL: /s_n_clone/social_networking/users/login
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(92): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 65)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-12-16 07:33:40 Error: [Facebook\FacebookSDKException] You must provide or set a default application id.
Request URL: /s_n_clone/social_networking/users/login?code=AQD-YwzahlbgJmYMi1EyspMLZg9JHbyxX8O7rChi4ZDpXQ0htEBGty9SoytBZpzyG5O2mnCp7gtX1JRiIkJDGKP1hqT81nfPUbx5vNLV5ecou_WsQcbmPG8cUpYwWd0kK-4bxcKzXXX3Gq-GYG52ewTBbHxc1ARGdVLD6fK03xf3IIBDPZeuGip17H0tZUlzfAAsDk5fVAZJfsz1WoWI6CoxOL5mFpZ3sdhCc4UcZlxIxY0-Qy1hiQx_Y7-HWxiEpjIRcVR8YpLVfHFkYED80SF0gjY4M2kEOszQyVGkE5YJ1akV__CmxVBANW4z4lUX0lU&state=812cd4064246ede91cb049f539e96441
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\facebook\php-sdk-v4\src\Facebook\FacebookRedirectLoginHelper.php(75): Facebook\FacebookSession::_getTargetAppId(NULL)
#1 C:\xampp\htdocs\s_n_clone\social_networking\vendor\akkaweb\cakephp-facebook\src\Controller\Component\GraphComponent.php(229): Facebook\FacebookRedirectLoginHelper->__construct('http://localhos...')
#2 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventManager.php(385): AkkaFacebook\Controller\Component\GraphComponent->beforeFilter(Object(Cake\Event\Event))
#3 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventManager.php(355): Cake\Event\EventManager->_callListener(Array, Object(Cake\Event\Event))
#4 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventDispatcherTrait.php(78): Cake\Event\EventManager->dispatch(Object(Cake\Event\Event))
#5 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Controller.php(491): Cake\Controller\Controller->dispatchEvent('Controller.init...')
#6 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(109): Cake\Controller\Controller->startupProcess()
#7 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#8 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#9 {main}


2015-12-16 10:11:48 Error: [Cake\Core\Exception\Exception] App\Controller\PagesController does not implement an isAuthorized() method.
Request URL: /s_n_clone/social_networking/
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Auth\ControllerAuthorize.php(58): Cake\Auth\ControllerAuthorize->controller(Object(App\Controller\PagesController))
#1 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Component\AuthComponent.php(535): Cake\Auth\ControllerAuthorize->__construct(Object(Cake\Controller\ComponentRegistry), Array)
#2 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Component\AuthComponent.php(491): Cake\Controller\Component\AuthComponent->constructAuthorize()
#3 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Component\AuthComponent.php(305): Cake\Controller\Component\AuthComponent->isAuthorized(Array)
#4 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Component\AuthComponent.php(258): Cake\Controller\Component\AuthComponent->authCheck(Object(Cake\Event\Event))
#5 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventManager.php(385): Cake\Controller\Component\AuthComponent->startup(Object(Cake\Event\Event))
#6 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventManager.php(355): Cake\Event\EventManager->_callListener(Array, Object(Cake\Event\Event))
#7 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventDispatcherTrait.php(78): Cake\Event\EventManager->dispatch(Object(Cake\Event\Event))
#8 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Controller.php(495): Cake\Controller\Controller->dispatchEvent('Controller.star...')
#9 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(109): Cake\Controller\Controller->startupProcess()
#10 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\PagesController))
#11 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#12 {main}


2015-12-16 10:12:26 Error: [Cake\Core\Exception\Exception] App\Controller\PagesController does not implement an isAuthorized() method.
Request URL: /s_n_clone/social_networking/
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Auth\ControllerAuthorize.php(58): Cake\Auth\ControllerAuthorize->controller(Object(App\Controller\PagesController))
#1 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Component\AuthComponent.php(535): Cake\Auth\ControllerAuthorize->__construct(Object(Cake\Controller\ComponentRegistry), Array)
#2 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Component\AuthComponent.php(491): Cake\Controller\Component\AuthComponent->constructAuthorize()
#3 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Component\AuthComponent.php(305): Cake\Controller\Component\AuthComponent->isAuthorized(Array)
#4 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Component\AuthComponent.php(258): Cake\Controller\Component\AuthComponent->authCheck(Object(Cake\Event\Event))
#5 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventManager.php(385): Cake\Controller\Component\AuthComponent->startup(Object(Cake\Event\Event))
#6 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventManager.php(355): Cake\Event\EventManager->_callListener(Array, Object(Cake\Event\Event))
#7 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventDispatcherTrait.php(78): Cake\Event\EventManager->dispatch(Object(Cake\Event\Event))
#8 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Controller.php(495): Cake\Controller\Controller->dispatchEvent('Controller.star...')
#9 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(109): Cake\Controller\Controller->startupProcess()
#10 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\PagesController))
#11 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#12 {main}


2015-12-16 10:14:33 Error: Fatal Error (1): Call to a member function user() on boolean in [C:\xampp\htdocs\s_n_clone\social_networking\src\Controller\AppController.php, line 89]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 209
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 92
[main] - [internal], line ??



2015-12-16 10:14:34 Error: [Cake\Error\FatalErrorException] Call to a member function user() on boolean
Request URL: /s_n_clone/social_networking/
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(92): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 89)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-12-16 10:15:12 Error: [Cake\Core\Exception\Exception] App\Controller\PagesController does not implement an isAuthorized() method.
Request URL: /s_n_clone/social_networking/
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Auth\ControllerAuthorize.php(58): Cake\Auth\ControllerAuthorize->controller(Object(App\Controller\PagesController))
#1 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Component\AuthComponent.php(535): Cake\Auth\ControllerAuthorize->__construct(Object(Cake\Controller\ComponentRegistry), Array)
#2 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Component\AuthComponent.php(491): Cake\Controller\Component\AuthComponent->constructAuthorize()
#3 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Component\AuthComponent.php(305): Cake\Controller\Component\AuthComponent->isAuthorized(Array)
#4 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Component\AuthComponent.php(258): Cake\Controller\Component\AuthComponent->authCheck(Object(Cake\Event\Event))
#5 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventManager.php(385): Cake\Controller\Component\AuthComponent->startup(Object(Cake\Event\Event))
#6 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventManager.php(355): Cake\Event\EventManager->_callListener(Array, Object(Cake\Event\Event))
#7 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Event\EventDispatcherTrait.php(78): Cake\Event\EventManager->dispatch(Object(Cake\Event\Event))
#8 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Controller.php(495): Cake\Controller\Controller->dispatchEvent('Controller.star...')
#9 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(109): Cake\Controller\Controller->startupProcess()
#10 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\PagesController))
#11 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#12 {main}


2015-12-16 10:33:44 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Logout could not be found.
Exception Attributes: array (
  'class' => 'Logout',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /s_n_clone/social_networking/logout
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-12-16 10:34:36 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Add could not be found.
Exception Attributes: array (
  'class' => 'Add',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /s_n_clone/social_networking/add
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-12-16 11:30:17 Warning: Warning (2): session_regenerate_id(): Session object destruction failed in [C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Network\Session.php, line 576]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 140
session_regenerate_id - [internal], line ??
Cake\Network\Session::renew() - CORE\src\Network\Session.php, line 576
Cake\Auth\Storage\SessionStorage::write() - CORE\src\Auth\Storage\SessionStorage.php, line 101
Cake\Controller\Component\AuthComponent::setUser() - CORE\src\Controller\Component\AuthComponent.php, line 631
App\Controller\UsersController::login() - APP/Controller\UsersController.php, line 53
Cake\Controller\Controller::invokeAction() - CORE\src\Controller\Controller.php, line 429
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 114
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-12-16 11:30:18 Warning: Warning (2): session_regenerate_id(): Session object destruction failed in [C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Network\Session.php, line 576]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 140
session_regenerate_id - [internal], line ??
Cake\Network\Session::renew() - CORE\src\Network\Session.php, line 576
Cake\Auth\Storage\SessionStorage::write() - CORE\src\Auth\Storage\SessionStorage.php, line 101
Cake\Controller\Component\AuthComponent::setUser() - CORE\src\Controller\Component\AuthComponent.php, line 631
App\Controller\UsersController::login() - APP/Controller\UsersController.php, line 53
Cake\Controller\Controller::invokeAction() - CORE\src\Controller\Controller.php, line 429
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 114
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-12-16 11:35:40 Error: Fatal Error (4): syntax error, unexpected ')' in [C:\xampp\htdocs\s_n_clone\social_networking\src\Controller\UsersController.php, line 90]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 209
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 92
[main] - [internal], line ??



2015-12-16 11:35:40 Error: [Cake\Error\FatalErrorException] syntax error, unexpected ')'
Request URL: /s_n_clone/social_networking/users/
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(92): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 90)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-12-16 11:36:42 Error: Fatal Error (1): Class 'App\Controller\Router' not found in [C:\xampp\htdocs\s_n_clone\social_networking\src\Controller\UsersController.php, line 90]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 209
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 92
[main] - [internal], line ??



2015-12-16 11:36:42 Error: [Cake\Error\FatalErrorException] Class 'App\Controller\Router' not found
Request URL: /s_n_clone/social_networking/users/
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(92): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Class 'App\\Cont...', 'C:\\xampp\\htdocs...', 90)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-12-16 11:38:08 Error: Fatal Error (4): syntax error, unexpected ';' in [C:\xampp\htdocs\s_n_clone\social_networking\src\Controller\UsersController.php, line 92]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 209
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 92
[main] - [internal], line ??



2015-12-16 11:38:09 Error: [Cake\Error\FatalErrorException] syntax error, unexpected ';'
Request URL: /s_n_clone/social_networking/users/
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(92): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 92)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-12-16 11:39:14 Error: [Cake\Datasource\Exception\InvalidPrimaryKeyException] Record not found in table "users" with primary key [NULL]
Request URL: /s_n_clone/social_networking/users/edit-user
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\src\Controller\UsersController.php(161): Cake\ORM\Table->get(NULL, Array)
#1 [internal function]: App\Controller\UsersController->editUser()
#2 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Controller.php(429): call_user_func_array(Array, Array)
#3 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#4 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#5 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#6 {main}


2015-12-16 11:39:31 Error: [Cake\Datasource\Exception\RecordNotFoundException] Record not found in table "users"
Request URL: /s_n_clone/social_networking/users/edit-user/1
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1187): Cake\ORM\Query->firstOrFail()
#1 C:\xampp\htdocs\s_n_clone\social_networking\src\Controller\UsersController.php(161): Cake\ORM\Table->get('1', Array)
#2 [internal function]: App\Controller\UsersController->editUser('1')
#3 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Controller.php(429): call_user_func_array(Array, Array)
#4 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#5 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#6 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#7 {main}


2015-12-16 11:39:43 Error: [Cake\View\Exception\MissingTemplateException] Template file "Users\edit_user.ctp" is missing.
Exception Attributes: array (
  'file' => 'Users\\edit_user.ctp',
)
Request URL: /s_n_clone/social_networking/users/edit-user/3
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\View\View.php(576): Cake\View\View->_getViewFileName(NULL)
#1 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Controller.php(611): Cake\View\View->render(NULL, NULL)
#2 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(120): Cake\Controller\Controller->render()
#3 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#4 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#5 {main}


2015-12-16 11:48:00 Error: [Cake\Datasource\Exception\InvalidPrimaryKeyException] Record not found in table "users" with primary key [NULL]
Request URL: /s_n_clone/social_networking/users/edit-user?id=3
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\src\Controller\UsersController.php(160): Cake\ORM\Table->get(NULL, Array)
#1 [internal function]: App\Controller\UsersController->editUser()
#2 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Controller.php(429): call_user_func_array(Array, Array)
#3 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#4 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#5 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#6 {main}


2015-12-16 11:48:38 Error: [Cake\Datasource\Exception\InvalidPrimaryKeyException] Record not found in table "users" with primary key [NULL]
Request URL: /s_n_clone/social_networking/users/edit-user?id=3
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\src\Controller\UsersController.php(160): Cake\ORM\Table->get(NULL, Array)
#1 [internal function]: App\Controller\UsersController->editUser()
#2 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Controller.php(429): call_user_func_array(Array, Array)
#3 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#4 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#5 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#6 {main}


2015-12-16 13:26:05 Error: [BadMethodCallException] Unknown method "saveAssociated"
Request URL: /s_n_clone/social_networking/users/signup
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\src\Controller\UsersController.php(57): Cake\ORM\Table->__call('saveAssociated', Array)
#1 C:\xampp\htdocs\s_n_clone\social_networking\src\Controller\UsersController.php(57): App\Model\Table\UsersTable->saveAssociated(Array, Array)
#2 [internal function]: App\Controller\UsersController->signup()
#3 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Controller.php(429): call_user_func_array(Array, Array)
#4 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#5 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#6 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#7 {main}


2015-12-16 13:27:32 Error: [BadMethodCallException] Unknown method "saveAssociated"
Request URL: /s_n_clone/social_networking/users/signup
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\src\Controller\UsersController.php(57): Cake\ORM\Table->__call('saveAssociated', Array)
#1 C:\xampp\htdocs\s_n_clone\social_networking\src\Controller\UsersController.php(57): App\Model\Table\UsersTable->saveAssociated(Array, Array)
#2 [internal function]: App\Controller\UsersController->signup()
#3 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Controller.php(429): call_user_func_array(Array, Array)
#4 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#5 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#6 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#7 {main}


2015-12-16 13:44:16 Error: [BadMethodCallException] Unknown method "saveAssociated"
Request URL: /s_n_clone/social_networking/users/signup
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\src\Controller\UsersController.php(57): Cake\ORM\Table->__call('saveAssociated', Array)
#1 C:\xampp\htdocs\s_n_clone\social_networking\src\Controller\UsersController.php(57): App\Model\Table\UsersTable->saveAssociated(Array, Array)
#2 [internal function]: App\Controller\UsersController->signup()
#3 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Controller.php(429): call_user_func_array(Array, Array)
#4 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#5 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#6 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#7 {main}


2015-12-16 13:45:52 Error: Fatal Error (1): Call to a member function saveAssociated() on boolean in [C:\xampp\htdocs\s_n_clone\social_networking\src\Controller\UsersController.php, line 57]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 209
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 92
[main] - [internal], line ??



2015-12-16 13:45:52 Error: [Cake\Error\FatalErrorException] Call to a member function saveAssociated() on boolean
Request URL: /s_n_clone/social_networking/users/signup
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(92): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 57)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-12-16 13:46:33 Error: [BadMethodCallException] Unknown method "saveAssociated"
Request URL: /s_n_clone/social_networking/users/signup
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\src\Controller\UsersController.php(57): Cake\ORM\Table->__call('saveAssociated', Array)
#1 C:\xampp\htdocs\s_n_clone\social_networking\src\Controller\UsersController.php(57): App\Model\Table\UsersTable->saveAssociated(Array, Array)
#2 [internal function]: App\Controller\UsersController->signup()
#3 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Controller.php(429): call_user_func_array(Array, Array)
#4 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#5 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#6 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#7 {main}


2015-12-16 13:47:11 Error: [BadMethodCallException] Unknown method "saveAssociated"
Request URL: /s_n_clone/social_networking/users/signup
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\src\Controller\UsersController.php(57): Cake\ORM\Table->__call('saveAssociated', Array)
#1 C:\xampp\htdocs\s_n_clone\social_networking\src\Controller\UsersController.php(57): App\Model\Table\UsersTable->saveAssociated(Array, Array)
#2 [internal function]: App\Controller\UsersController->signup()
#3 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Controller.php(429): call_user_func_array(Array, Array)
#4 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#5 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#6 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#7 {main}


2015-12-16 13:48:48 Error: [BadMethodCallException] Unknown method "saveMany"
Request URL: /s_n_clone/social_networking/users/signup
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\src\Controller\UsersController.php(57): Cake\ORM\Table->__call('saveMany', Array)
#1 C:\xampp\htdocs\s_n_clone\social_networking\src\Controller\UsersController.php(57): App\Model\Table\UsersTable->saveMany(Array, Array)
#2 [internal function]: App\Controller\UsersController->signup()
#3 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Controller.php(429): call_user_func_array(Array, Array)
#4 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#5 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#6 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#7 {main}


2015-12-16 14:03:25 Warning: Warning (4096): Argument 1 passed to Cake\ORM\Table::save() must implement interface Cake\Datasource\EntityInterface, array given, called in C:\xampp\htdocs\s_n_clone\social_networking\src\Controller\UsersController.php on line 55 and defined in [C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php, line 1356]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 140
Cake\ORM\Table::save() - CORE\src\ORM\Table.php, line 1356
App\Controller\UsersController::signup() - APP/Controller\UsersController.php, line 55
Cake\Controller\Controller::invokeAction() - CORE\src\Controller\Controller.php, line 429
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 114
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-12-16 14:03:25 Error: Fatal Error (1): Call to a member function errors() on array in [C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php, line 1366]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 209
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 92
[main] - [internal], line ??



2015-12-16 14:03:26 Error: [Cake\Error\FatalErrorException] Call to a member function errors() on array
Request URL: /s_n_clone/social_networking/users/signup
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(92): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 1366)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-12-16 14:05:30 Warning: Warning (4096): Argument 1 passed to Cake\ORM\Table::save() must implement interface Cake\Datasource\EntityInterface, array given, called in C:\xampp\htdocs\s_n_clone\social_networking\src\Controller\UsersController.php on line 56 and defined in [C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php, line 1356]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 140
Cake\ORM\Table::save() - CORE\src\ORM\Table.php, line 1356
App\Controller\UsersController::signup() - APP/Controller\UsersController.php, line 56
Cake\Controller\Controller::invokeAction() - CORE\src\Controller\Controller.php, line 429
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 114
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-12-16 14:05:30 Error: Fatal Error (1): Call to a member function errors() on array in [C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php, line 1366]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 209
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 92
[main] - [internal], line ??



2015-12-16 14:05:30 Error: [Cake\Error\FatalErrorException] Call to a member function errors() on array
Request URL: /s_n_clone/social_networking/users/signup
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(92): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 1366)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-12-16 14:06:56 Error: Fatal Error (1): Class 'App\Controller\TableRegistry' not found in [C:\xampp\htdocs\s_n_clone\social_networking\src\Controller\UsersController.php, line 49]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 209
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 92
[main] - [internal], line ??



2015-12-16 14:06:56 Error: [Cake\Error\FatalErrorException] Class 'App\Controller\TableRegistry' not found
Request URL: /s_n_clone/social_networking/users/signup
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(92): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Class 'App\\Cont...', 'C:\\xampp\\htdocs...', 49)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-12-16 14:41:39 Error: [InvalidArgumentException] Could not save profiles, it cannot be traversed
Request URL: /s_n_clone/social_networking/users/signup
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\AssociationCollection.php(251): Cake\ORM\Association\HasMany->saveAssociated(Object(App\Model\Entity\User), Array)
#1 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\AssociationCollection.php(227): Cake\ORM\AssociationCollection->_save(Object(Cake\ORM\Association\HasMany), Object(App\Model\Entity\User), Array, Array)
#2 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\AssociationCollection.php(192): Cake\ORM\AssociationCollection->_saveAssociations(Object(App\Model\Table\UsersTable), Object(App\Model\Entity\User), Array, Array, true)
#3 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1457): Cake\ORM\AssociationCollection->saveChildren(Object(App\Model\Table\UsersTable), Object(App\Model\Entity\User), Array, Array)
#4 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1377): Cake\ORM\Table->_processSave(Object(App\Model\Entity\User), Object(ArrayObject))
#5 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Database\Connection.php(557): Cake\ORM\Table->Cake\ORM\{closure}(Object(Cake\Database\Connection))
#6 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1378): Cake\Database\Connection->transactional(Object(Closure))
#7 C:\xampp\htdocs\s_n_clone\social_networking\src\Controller\UsersController.php(61): Cake\ORM\Table->save(Object(App\Model\Entity\User))
#8 [internal function]: App\Controller\UsersController->signup()
#9 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Controller.php(429): call_user_func_array(Array, Array)
#10 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#11 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#12 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#13 {main}


2015-12-16 14:45:51 Error: [InvalidArgumentException] Could not save profiles, it cannot be traversed
Request URL: /s_n_clone/social_networking/users/signup
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\AssociationCollection.php(251): Cake\ORM\Association\HasMany->saveAssociated(Object(App\Model\Entity\User), Array)
#1 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\AssociationCollection.php(227): Cake\ORM\AssociationCollection->_save(Object(Cake\ORM\Association\HasMany), Object(App\Model\Entity\User), Array, Array)
#2 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\AssociationCollection.php(192): Cake\ORM\AssociationCollection->_saveAssociations(Object(App\Model\Table\UsersTable), Object(App\Model\Entity\User), Array, Array, true)
#3 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1457): Cake\ORM\AssociationCollection->saveChildren(Object(App\Model\Table\UsersTable), Object(App\Model\Entity\User), Array, Array)
#4 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1377): Cake\ORM\Table->_processSave(Object(App\Model\Entity\User), Object(ArrayObject))
#5 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Database\Connection.php(557): Cake\ORM\Table->Cake\ORM\{closure}(Object(Cake\Database\Connection))
#6 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1378): Cake\Database\Connection->transactional(Object(Closure))
#7 C:\xampp\htdocs\s_n_clone\social_networking\src\Controller\UsersController.php(62): Cake\ORM\Table->save(Object(App\Model\Entity\User))
#8 [internal function]: App\Controller\UsersController->signup()
#9 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Controller.php(429): call_user_func_array(Array, Array)
#10 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#11 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#12 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#13 {main}


2015-12-16 15:50:37 Error: [InvalidArgumentException] Could not save profiles, it cannot be traversed
Request URL: /s_n_clone/social_networking/users/signup
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\AssociationCollection.php(251): Cake\ORM\Association\HasMany->saveAssociated(Object(App\Model\Entity\User), Array)
#1 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\AssociationCollection.php(227): Cake\ORM\AssociationCollection->_save(Object(Cake\ORM\Association\HasMany), Object(App\Model\Entity\User), Array, Array)
#2 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\AssociationCollection.php(192): Cake\ORM\AssociationCollection->_saveAssociations(Object(App\Model\Table\UsersTable), Object(App\Model\Entity\User), Array, Array, true)
#3 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1457): Cake\ORM\AssociationCollection->saveChildren(Object(App\Model\Table\UsersTable), Object(App\Model\Entity\User), Array, Array)
#4 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1377): Cake\ORM\Table->_processSave(Object(App\Model\Entity\User), Object(ArrayObject))
#5 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Database\Connection.php(557): Cake\ORM\Table->Cake\ORM\{closure}(Object(Cake\Database\Connection))
#6 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1378): Cake\Database\Connection->transactional(Object(Closure))
#7 C:\xampp\htdocs\s_n_clone\social_networking\src\Controller\UsersController.php(62): Cake\ORM\Table->save(Object(App\Model\Entity\User))
#8 [internal function]: App\Controller\UsersController->signup()
#9 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Controller.php(429): call_user_func_array(Array, Array)
#10 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#11 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#12 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#13 {main}


2015-12-16 15:54:18 Error: [InvalidArgumentException] Could not save profiles, it cannot be traversed
Request URL: /s_n_clone/social_networking/users/signup
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\AssociationCollection.php(251): Cake\ORM\Association\HasMany->saveAssociated(Object(App\Model\Entity\User), Array)
#1 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\AssociationCollection.php(227): Cake\ORM\AssociationCollection->_save(Object(Cake\ORM\Association\HasMany), Object(App\Model\Entity\User), Array, Array)
#2 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\AssociationCollection.php(192): Cake\ORM\AssociationCollection->_saveAssociations(Object(App\Model\Table\UsersTable), Object(App\Model\Entity\User), Array, Array, true)
#3 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1457): Cake\ORM\AssociationCollection->saveChildren(Object(App\Model\Table\UsersTable), Object(App\Model\Entity\User), Array, Array)
#4 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1377): Cake\ORM\Table->_processSave(Object(App\Model\Entity\User), Object(ArrayObject))
#5 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Database\Connection.php(557): Cake\ORM\Table->Cake\ORM\{closure}(Object(Cake\Database\Connection))
#6 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1378): Cake\Database\Connection->transactional(Object(Closure))
#7 C:\xampp\htdocs\s_n_clone\social_networking\src\Controller\UsersController.php(62): Cake\ORM\Table->save(Object(App\Model\Entity\User))
#8 [internal function]: App\Controller\UsersController->signup()
#9 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Controller.php(429): call_user_func_array(Array, Array)
#10 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#11 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#12 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#13 {main}


2015-12-16 16:01:55 Error: [InvalidArgumentException] Could not save profiles, it cannot be traversed
Request URL: /s_n_clone/social_networking/users/signup
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\AssociationCollection.php(251): Cake\ORM\Association\HasMany->saveAssociated(Object(App\Model\Entity\User), Array)
#1 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\AssociationCollection.php(227): Cake\ORM\AssociationCollection->_save(Object(Cake\ORM\Association\HasMany), Object(App\Model\Entity\User), Array, Array)
#2 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\AssociationCollection.php(192): Cake\ORM\AssociationCollection->_saveAssociations(Object(App\Model\Table\UsersTable), Object(App\Model\Entity\User), Array, Array, true)
#3 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1457): Cake\ORM\AssociationCollection->saveChildren(Object(App\Model\Table\UsersTable), Object(App\Model\Entity\User), Array, Array)
#4 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1377): Cake\ORM\Table->_processSave(Object(App\Model\Entity\User), Object(ArrayObject))
#5 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Database\Connection.php(557): Cake\ORM\Table->Cake\ORM\{closure}(Object(Cake\Database\Connection))
#6 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1378): Cake\Database\Connection->transactional(Object(Closure))
#7 C:\xampp\htdocs\s_n_clone\social_networking\src\Controller\UsersController.php(63): Cake\ORM\Table->save(Object(App\Model\Entity\User))
#8 [internal function]: App\Controller\UsersController->signup()
#9 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Controller.php(429): call_user_func_array(Array, Array)
#10 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#11 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#12 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#13 {main}


2015-12-16 16:02:37 Error: [InvalidArgumentException] Could not save profiles, it cannot be traversed
Request URL: /s_n_clone/social_networking/users/signup
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\AssociationCollection.php(251): Cake\ORM\Association\HasMany->saveAssociated(Object(App\Model\Entity\User), Array)
#1 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\AssociationCollection.php(227): Cake\ORM\AssociationCollection->_save(Object(Cake\ORM\Association\HasMany), Object(App\Model\Entity\User), Array, Array)
#2 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\AssociationCollection.php(192): Cake\ORM\AssociationCollection->_saveAssociations(Object(App\Model\Table\UsersTable), Object(App\Model\Entity\User), Array, Array, true)
#3 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1457): Cake\ORM\AssociationCollection->saveChildren(Object(App\Model\Table\UsersTable), Object(App\Model\Entity\User), Array, Array)
#4 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1377): Cake\ORM\Table->_processSave(Object(App\Model\Entity\User), Object(ArrayObject))
#5 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Database\Connection.php(557): Cake\ORM\Table->Cake\ORM\{closure}(Object(Cake\Database\Connection))
#6 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1378): Cake\Database\Connection->transactional(Object(Closure))
#7 C:\xampp\htdocs\s_n_clone\social_networking\src\Controller\UsersController.php(63): Cake\ORM\Table->save(Object(App\Model\Entity\User))
#8 [internal function]: App\Controller\UsersController->signup()
#9 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Controller.php(429): call_user_func_array(Array, Array)
#10 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#11 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#12 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#13 {main}


2015-12-16 16:42:15 Error: Fatal Error (1): Class 'AuthComponent' not found in [C:\xampp\htdocs\s_n_clone\social_networking\src\Template\Posts\add_post.ctp, line 12]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 209
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 92
[main] - [internal], line ??



2015-12-16 16:42:16 Error: [Cake\Error\FatalErrorException] Class 'AuthComponent' not found
Request URL: /s_n_clone/social_networking/posts/add-post
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(92): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Class 'AuthComp...', 'C:\\xampp\\htdocs...', 12)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-12-16 16:43:53 Error: Fatal Error (1): Class 'Router' not found in [C:\xampp\htdocs\s_n_clone\social_networking\src\Template\Posts\add_post.ctp, line 12]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 209
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 92
[main] - [internal], line ??



2015-12-16 16:43:53 Error: [Cake\Error\FatalErrorException] Class 'Router' not found
Request URL: /s_n_clone/social_networking/posts/add-post
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(92): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Class 'Router' ...', 'C:\\xampp\\htdocs...', 12)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-12-16 16:48:50 Error: Fatal Error (1): Call to a member function read() on null in [C:\xampp\htdocs\s_n_clone\social_networking\src\Template\Posts\add_post.ctp, line 12]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 209
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 92
[main] - [internal], line ??



2015-12-16 16:48:51 Error: [Cake\Error\FatalErrorException] Call to a member function read() on null
Request URL: /s_n_clone/social_networking/posts/add-post
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(92): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 12)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-12-16 17:16:27 Error: [Cake\Datasource\Exception\RecordNotFoundException] Record not found in table "posts"
Request URL: /s_n_clone/social_networking/posts/view-post/1
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1187): Cake\ORM\Query->firstOrFail()
#1 C:\xampp\htdocs\s_n_clone\social_networking\src\Controller\PostsController.php(66): Cake\ORM\Table->get('1', Array)
#2 [internal function]: App\Controller\PostsController->viewPost('1')
#3 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Controller.php(429): call_user_func_array(Array, Array)
#4 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#5 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\PostsController))
#6 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#7 {main}


2015-12-16 17:30:18 Error: [Cake\Controller\Exception\MissingActionException] Action PostsController::editUser() could not be found, or is not accessible.
Exception Attributes: array (
  'controller' => 'PostsController',
  'action' => 'editUser',
  'prefix' => '',
  'plugin' => NULL,
)
Request URL: /s_n_clone/social_networking/posts/edit-user/23
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#1 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\PostsController))
#2 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 {main}


2015-12-16 17:31:01 Error: [Cake\Controller\Exception\MissingActionException] Action PostsController::editUser() could not be found, or is not accessible.
Exception Attributes: array (
  'controller' => 'PostsController',
  'action' => 'editUser',
  'prefix' => '',
  'plugin' => NULL,
)
Request URL: /s_n_clone/social_networking/posts/edit-user/23
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#1 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\PostsController))
#2 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 {main}


2015-12-16 17:56:13 Warning: Warning (2): Missing argument 1 for App\Controller\CommentsController::addComment() in [C:\xampp\htdocs\s_n_clone\social_networking\src\Controller\CommentsController.php, line 75]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 140
App\Controller\CommentsController::addComment() - APP/Controller\CommentsController.php, line 75
Cake\Controller\Controller::invokeAction() - CORE\src\Controller\Controller.php, line 429
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 114
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-12-16 17:59:22 Error: [Cake\Controller\Exception\MissingActionException] Action CommentsController::indexPost() could not be found, or is not accessible.
Exception Attributes: array (
  'controller' => 'CommentsController',
  'action' => 'indexPost',
  'prefix' => '',
  'plugin' => NULL,
)
Request URL: /s_n_clone/social_networking/comments/index-post
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#1 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\CommentsController))
#2 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 {main}


2015-12-16 17:59:57 Error: [Cake\Controller\Exception\MissingActionException] Action CommentsController::indexPost() could not be found, or is not accessible.
Exception Attributes: array (
  'controller' => 'CommentsController',
  'action' => 'indexPost',
  'prefix' => '',
  'plugin' => NULL,
)
Request URL: /s_n_clone/social_networking/comments/index-post
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#1 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\CommentsController))
#2 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 {main}


2015-12-16 18:09:31 Error: [PDOException] SQLSTATE[23000]: Integrity constraint violation: 1451 Cannot delete or update a parent row: a foreign key constraint fails (`social_networking`.`comments`, CONSTRAINT `comment_post_key` FOREIGN KEY (`post_id`) REFERENCES `posts` (`id`) ON DELETE NO ACTION ON UPDATE NO ACTION)
Request URL: /s_n_clone/social_networking/posts/delete-post/2
Stack Trace:
#0 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Database\Statement\MysqlStatement.php(36): PDOStatement->execute(NULL)
#1 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Database\Statement\StatementDecorator.php(171): Cake\Database\Statement\MysqlStatement->execute(NULL)
#2 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Database\Log\LoggingStatement.php(56): Cake\Database\Statement\StatementDecorator->execute(NULL)
#3 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Database\Connection.php(270): Cake\Database\Log\LoggingStatement->execute()
#4 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Database\Query.php(174): Cake\Database\Connection->run(Object(Cake\ORM\Query))
#5 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1711): Cake\Database\Query->execute()
#6 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1643): Cake\ORM\Table->_processDelete(Object(App\Model\Entity\Post), Object(ArrayObject))
#7 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Database\Connection.php(557): Cake\ORM\Table->Cake\ORM\{closure}(Object(Cake\Database\Connection))
#8 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\ORM\Table.php(1648): Cake\Database\Connection->transactional(Object(Closure))
#9 C:\xampp\htdocs\s_n_clone\social_networking\src\Controller\PostsController.php(159): Cake\ORM\Table->delete(Object(App\Model\Entity\Post))
#10 [internal function]: App\Controller\PostsController->deletePost('2')
#11 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Controller\Controller.php(429): call_user_func_array(Array, Array)
#12 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#13 C:\xampp\htdocs\s_n_clone\social_networking\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\PostsController))
#14 C:\xampp\htdocs\s_n_clone\social_networking\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#15 {main}


